plugin/src/main/java/io/jenkins/plugins/redis/RedisFingerprintStorage.java

```diff
@@ -55,36 +57,19 @@
  */
 
 @Extension
+@Symbol("redis")
```

Comment:
Might be an issue for JCasC if there are other cobtrols with the same name. Anyway, it is not a problem for this plugin. It is a JCasC/symbols architecture issue

Reply:
jcasc takes into account the extension point so fine as far as I know

Reply:
Ill remove it nevertheless.

Reply:
I don’t think it needs removing

Reply:
if by default it is being inferred as `redis`, I am not sure why leave it